docs/crio.8.md

```diff
@@ -90,196 +100,225 @@ crio is meant to provide an integration path between OCI conformant runtimes and
 6. Resource isolation as required by the CRI.
 
 **Usage**:
+
 ```
-crio [GLOBAL OPTIONS]
-crio [GLOBAL OPTIONS] config [OPTIONS]
+crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 ```
+
 # GLOBAL OPTIONS
-**--additional-devices**="": devices to add to the containers
 
-**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is "crio-default-" followed by the version string of CRI-O.
+**--additional-devices**="": Devices to add to the containers (default: [])
+
+**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is 'crio-default-' followed by the version string of CRI-O. (default: "crio-default-1.17.0-dev")
 
-**--bind-mount-prefix**="": A prefix to use for the source of the bind mounts.  This option would be useful if you were running CRI-O in a container.  And had `/` mounted on `/host` in your container.  Then if you ran CRI-O with the `--bind-mount-prefix=/host` option, CRI-O would add /host to any bind mounts it is handed over CRI.  If Kubernetes asked to have `/var/lib/foobar` bind mounted into the container, then CRI-O would bind mount `/host/var/lib/foobar`.  Since CRI-O itself is running in a container with `/` or the host mounted on `/host`, the container would end up with `/var/lib/foobar` from the host mounted in the container rather then `/var/lib/foobar` from the CRI-O container.
+**--bind-mount-prefix**="": A prefix to use for the source of the bind mounts. This option would be useful if you were running CRI-O in a container. And had `/` mounted on `/host` in your container. Then if you ran CRI-O with the `--bind-mount-prefix=/host` option, CRI-O would add /host to any bind mounts it is handed over CRI. If Kubernetes asked to have `/var/lib/foobar` bind mounted into the container, then CRI-O would bind mount `/host/var/lib/foobar`. Since CRI-O itself is running in a container with `/` or the host mounted on `/host`, the container would end up with `/var/lib/foobar` from the host mounted in the container rather then `/var/lib/foobar` from the CRI-O container. (default: "")
 
-**--cgroup-manager**="": cgroup manager (cgroupfs or systemd)
+**--cgroup-manager**="": Cgroup manager (cgroupfs or systemd) (default: "cgroupfs")
 
 **--cni-config-dir**="": CNI configuration files directory (default: "/etc/cni/net.d/")
 
-**--cni-plugin-dir**="": CNI plugin binaries directory (default: "/opt/cni/bin/")
+**--cni-plugin-dir**="": CNI plugin binaries directory (default: "")
 
-**--config, -c**="": path to configuration file
+**--config, -c**="": Path to configuration file (default: /etc/crio/crio.conf)
 
 **--conmon**="": Path to the conmon binary, used for monitoring the OCI runtime. Will be searched for using $PATH if empty. (default: "")
 
-**--conmon-cgroup**="": cgroup used for conmon process (default: "pod")
+**--conmon-cgroup**="": Cgroup to be used for conmon process (default: "system.slice")
```

Comment:
Double check me on this one, but I think Cgroup is a weird one that's supposed to be lower cased 'cgroup'

Reply:
:+1: